solr/core/src/test/org/apache/solr/cloud/TestRandomFlRTGCloud.java

```diff
@@ -147,6 +148,7 @@ public class TestRandomFlRTGCloud extends SolrCloudTestCase {
           new NotIncludedValidator("score", "score_alias:score"));
 
   @BeforeClass
+  @SuppressWarnings({"rawtypes", "unchecked"})
```

Comment:
why?

Reply:
okay, looks like i got the right magic incantation.....